src/providers/google_live.py

```
@@ -1254,6 +1254,41 @@ async def _send_greeting(self) -> None:
+                "turnComplete": True,
+            }
+        }
+        try:
+            await self._send_message(message)
+            logger.info(
+                "Sent no-input announcement request to Google Live",
+                call_id=self._call_id,
+                text_preview=text[:80],
+            )
+            return True
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)